### PR TITLE
added missing properties to payment receipt

### PR DESCRIPTION
**Pull request by DFSko**

### CODE REVIEW

File: Api/Services/Payments/PaymentNotificationService.cs [hunk 1]
```diff
@@ -34,7 +36,23 @@ public Task<Result> SendReceiptToCustomer((DocumentRegistrationInfo Registration
                 Method = EnumFormatters.FromDescription(paymentReceipt.Method),
                 InvoiceNumber = paymentReceipt.InvoiceInfo.Number,
                 InvoiceDate = FormatDate(paymentReceipt.InvoiceInfo.Date),
-                ReferenceCode = paymentReceipt.ReferenceCode
+                ReferenceCode = paymentReceipt.ReferenceCode,
+                AccommodationName = paymentReceipt.AccommodationName,
+                RoomTypes = paymentReceipt.ReceiptItems.Select(r => new PaymentReceiptData.RoomDetails
```
Comment (mingazhev) [inline Api/Services/Payments/PaymentNotificationService.cs:41]: Please use RoomDetails as it is done in voucher. I've changed the documentation

File: Api/Services/Payments/PaymentNotificationService.cs [hunk 2]
```diff
@@ -34,7 +36,23 @@ public Task<Result> SendReceiptToCustomer((DocumentRegistrationInfo Registration
                 Method = EnumFormatters.FromDescription(paymentReceipt.Method),
                 InvoiceNumber = paymentReceipt.InvoiceInfo.Number,
```
Comment (mingazhev) [inline Api/Services/Payments/PaymentNotificationService.cs:37]: Please move this method to BookingMailingService, this service looks like having incorrect name.
We'll split BookingMailingService to separate service later

Comment (DFSko) [reply Api/Services/Payments/PaymentNotificationService.cs]: Fixed

File: Api/Services/Payments/PaymentNotificationService.cs (hunk 1 above)
Comment (DFSko) [reply Api/Services/Payments/PaymentNotificationService.cs]: Fixed

**Review by mingazhev — APPROVED** [review]

**Pull request merged**